CONTRIBUTOR.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to BOM Squad
+
+We welcome contributions to the BOM Squad project! By participating in this project, you agree to abide by the [Code of Conduct](code-of-conduct.md).
```

Comment:
`bom-squad` is the GitHub oganization. `protobom` is the specific project.

Reply:
Done